### ISSUE TITLE
Record Creation Date Stamp

### ISSUE THREAD

**Issue by emjohnst** [bcgov/BCHeritage#766]
Need to add uneditable but searchable creation date stamp to records, perhaps under site record admin?

Update: ability to search on edit log history

**Comment by bferguso** [comment]
Was thinking about making the edit log searchable rather than adding an audit-type field directly. Would that satisfy the requirement or would you prefer to have a creation date. The limitation on putting a field like this in a tile like Record Admin is that it wouldn't be created/populated until after that card is created, so if you created a Site, added a Borden Number then waited a day before creating the Record Admin, the date would reflect the date the Record Admin card was created, not when the Site was created.

Would the following page, if we were to make the fields more searchable, satisfy what you need? 
https://testapps.nrs.gov.bc.ca/int/bcrhp/resource/history

Not sure about how this could be done yet but looking into it.

If we do want to have the date visible somewhere in the Heritage Site model, could look in how we might apply this: https://timonweb.com/django/automatic-created-and-updated-datetime-fields-for-django-models/

**Comment by emjohnst** [comment]
@bferguso Yes, searchable edit log would be more ideal as I agree with the issues you've identified, it would be better to avoid that.

and Yes having the date/edit visible would be nice - can that be done post-mvp?
